tests/optimize/test_lie_algebra.py

```diff
@@ -189,14 +189,13 @@ def lie_circuit():
     hamiltonian_np = qml.utils.sparse_hamiltonian(hamiltonian, wires).toarray()
     lie_algebra_np = hamiltonian_np @ rho - rho @ hamiltonian_np
 
-    phi_exact = expm(-0.001 * lie_algebra_np) @ phi
+    phi_exact = expm(-0.1 * lie_algebra_np * 2**nqubits) @ phi
     rho_exact = np.outer(phi_exact, phi_exact.conj())
-    opt = LieAlgebraOptimizer(circuit=lie_circuit, stepsize=0.001, exact=True)
-    opt.step()
-
+    opt = LieAlgebraOptimizer(circuit=lie_circuit, stepsize=0.1, exact=True)
```

Comment:
Also curious why the stepsize changed here :)

Reply:
Larger stepsize means we need to be more precision, so this is a more sensitive test.